frontend/express/public/javascripts/countly/vue/core.js

```diff
@@ -95,7 +95,8 @@
                 state: {
                     period: countlyCommon.getPeriod(),
                     periodLabel: countlyCommon.getDateRangeForCalendar(),
-                    activeApp: null
+                    activeApp: null,
+                    allApps: []
```

Comment:
you need to initialise all apps here. it shouldn't be empty.
`allApps: countlyGlobal.apps`